loopy/target/c/codegen/expression.py

```diff
@@ -78,6 +78,8 @@ def __init__(self, codegen_state, fortran_abi=False, type_inf_mapper=None):
 
         self.fortran_abi = fortran_abi
 
+        self.allow_fp_reordering = self.kernel.options.allow_fp_reordering
```

Comment:
I'd prefer if we didn't do this. Just always use the full path, even if it's a bit more typing.

Reply:
Fixed in 2828501f455982e953f79d4c705d83b159a6a987